lib/collection/property-base.js

```diff
@@ -24,7 +24,7 @@ PropertyBase = function PropertyBase (definition) {
     // stored. more details on that can be retrieved from the propertyExtractMeta function itself.
     // @todo: make this a closed function to do getter and setter which is non enumerable
     var src = definition && definition.info || definition,
-        meta = _(src).pick(PropertyBase.propertyIsMeta).mapKeys(PropertyBase.propertyUnprefixMeta).value();
+        meta = _(src).pickBy(PropertyBase.propertyIsMeta).mapKeys(PropertyBase.propertyUnprefixMeta).value();
```

Comment:
Lodash `pick` does not accept a customizer function, that has been deferred to `pickBy`.